### PR TITLE
Issue 4462 | Intake Stats | Add average response time by notice date

### PR DESCRIPTION
**Pull request by ferlatte**

### CODE REVIEW

File: app/models/intake_stats.rb [hunk 1]
```diff
@@ -46,13 +46,12 @@ def cache_key
...
-        :response_time,
-        RampElection.completed.where(receipt_date: offset_range(range)),
-        50
-      )
+    # Average days to respond to RAMP election notice
+    average_election_response_time: lambda do |range|
+      elections = RampElection.completed.where(receipt_date: offset_range(range))
+      sum = 0.0
+      elections.each{ |e| sum += e.receipt_date.to_time.to_f - e.notice_date.to_time.to_f }
+      sum / elections.count.to_f
```
Comment (shanear) [inline app/models/intake_stats.rb]: We can probably turn this into an `average` function, bonus points, but not super necessary for this PR.

File: app/views/intake_stats/show.html.erb [hunk 2]
```diff
@@ -70,12 +70,12 @@
         </div>
       </div>
 
-      <div class="cf-stat-panel data-chart" data-key="median_election_response_time">
+      <div class="cf-stat-panel data-chart" data-key="average_election_response_time">
         <h4 class="cf-stat-title">
-          Median Response Time
+            Average Response Time
```
Comment (shanear) [inline app/views/intake_stats/show.html.erb]: extra leading spaces?

Comment (ferlatte) [reply app/views/intake_stats/show.html.erb]: Yep. Snuck in before I switched my editor to match the project. I'll fix this.

File: app/models/intake_stats.rb (hunk 1 above)
Comment (ferlatte) [reply app/models/intake_stats.rb]: I was hoping you were going to tell me that I had just missed the average method somewhere. ;)

**Pull request merged**